scripts/qiita

```diff
@@ -289,6 +294,49 @@ def update_preprocessed_data(sl_out_dir, study,  preprocessed_data):
                                             preprocessed_data)
     click.echo("Preprocessed data %s successfully updated" % ppd.id)
 
+# #############################################################################
+# PORTAL COMMANDS
+# #############################################################################
+
+
+@portal.command(name="add-analyses")
+@click.argument('portal', required=True, type=str)
```

Comment:
Just realized ... this argument should be a Choice type, because we know the name of all the portals and it should really only be one of those available.